### ISSUE TITLE
Could ExCo clone the current view to other view?

### ISSUE THREAD

**Issue by Mr-ZBin** [matkuki/ExCo#47]
Hi,

I know that Notepad++ can split the view of the same file with multiple views such as in left pane and in right pane.

![image](https://user-images.githubusercontent.com/22544491/168222802-0c11b96f-4b41-44fc-9d04-3ccbbb819cbf.png)

Thanks

**Comment by Mr-ZBin** [comment]
I know ExCo can copy view to other window, but it can't display text synchronously when I changed the content.

![image](https://user-images.githubusercontent.com/22544491/168223762-ef24e335-d8f6-4515-9711-c2091fc4098e.png)

**Comment by matkuki** [comment]
Hi @Mr-ZBin ,

If I understand correctly, you want a split view with the same file that updates the content of either view if you change the content of one, correct?
Can I ask for what purpose would you need two synchronized views to the same file?

**Comment by Mr-ZBin** [comment]
Hi @matkuki ,

That's totally correct. Sometimes I need to view the different areas in a same file to get some definition details for the variable to make sure what should I change about it. I have tried to create multiple views by [https://www.scintilla.org/ScintillaDoc.html#MultipleViews](https://www.scintilla.org/ScintillaDoc.html#MultipleViews), but I don't create the it successfully, and perhaps I don't use the guide correctly.
I did't found any other informations about this from google or bing.

Thank you very much.
